refinedstorage2-platform-forge/src/main/java/com/refinedmods/refinedstorage2/platform/forge/datagen/ItemModelProvider.java

```diff
@@ -0,0 +1,87 @@
+package com.refinedmods.refinedstorage2.platform.forge.datagen;
+
+import com.refinedmods.refinedstorage2.platform.common.content.BlockColorMap;
+import com.refinedmods.refinedstorage2.platform.common.content.Blocks;
+import com.refinedmods.refinedstorage2.platform.common.content.ColorMap;
+
+import net.minecraft.data.PackOutput;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.client.model.generators.ModelFile;
+import net.minecraftforge.common.data.ExistingFileHelper;
+
+import static com.refinedmods.refinedstorage2.platform.common.util.IdentifierUtil.MOD_ID;
+import static com.refinedmods.refinedstorage2.platform.common.util.IdentifierUtil.createIdentifier;
+
+public class ItemModelProvider extends net.minecraftforge.client.model.generators.ItemModelProvider {
+    public ItemModelProvider(final PackOutput output, final ExistingFileHelper existingFileHelper) {
+        super(output, MOD_ID, existingFileHelper);
+    }
+
+    @Override
+    protected void registerModels() {
+        registerCables();
+        registerController();
+        registerCreativeController();
+        registerGrid();
+    }
+
+    private void registerCables() {
+        final ResourceLocation base = createIdentifier("item/cable/base");
+        final ColorMap<?> cables = Blocks.INSTANCE.getCable();
```

Comment:
Why remove the typing here?
```suggestion
        final ColorMap<CableBlock> cables = Blocks.INSTANCE.getCable();
```